### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @gkoz

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[package]
+name = "gdk-pixbuf"
+version = "0.3.0"
```
Comment (gkoz) [inline Cargo.toml]: I think it's fine to start from `0.0.1`

**Comment by gkoz** [comment]
The commit message should probably mention the original repo and the splitting off

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by gkoz** [comment]
Alright, thanks. The rest of metadata we can adjusted separately.

**Pull request merged**